server/utils/allAPTypes.ts

```diff
@@ -0,0 +1,8 @@
+import { ApType } from '../@types/shared'
+
+export const allApTypes = {
```

Comment:
best to type rather than cast: `export const allApTypes: Record<ApType, string> = `